pages/api/profile/[user_id].js

```diff
@@ -1,20 +1,17 @@
 import moment from "moment";
 import { RequestHandler } from "../../../lib/RequestHandler";
 import { UPDATE_PROFILE_SCHEMA } from "../../../yup-schema/user";
+import { BadRequestException } from "../../../lib/BadRequestExcpetion";
 
 async function handle(req, res, prisma, user) {
   const { user_id } = req.query;
-
   const { organization_id } = user;
-
   const currentMonth = {
     lte: moment().endOf("month").format(),
     gte: moment().startOf("month").format(),
   };
 
-  if (!user_id) {
-    return res.status(401).json({ status: 401, message: "No User found" });
-  }
+  if (!user_id) throw BadRequestException("No User found");
 
   if (req.method === "GET") {
     const whereCondition = {
```

Comment:
use proper naming for this variable

Reply:
what are you asking to change "BadRequestException" 
as we have created a common exception for all the Bad request and sending just a string to it

Reply:
const whereCondition = { this varible name is not suitable